### ISSUE TITLE
When running inadyn as unprivileged user, it will complain about not able to chown the cache-dir. 

### ISSUE THREAD

**Issue by leonyu** [troglobit/inadyn#309]
When running inadyn as unprivileged user, it will complain about not able to `chown` the cache-dir. 

> Cannot change owner of cache directory /tmp/.inadyn/ to 0:0, skipping: Operation not permitted

Can be produced with the following docker/docker-compose setup: 

```yaml
services:
  inadyn:
    image: troglobit/inadyn:latest
    entrypoint: inadyn --foreground --cache-dir=/tmp/.inadyn/ --no-pidfile
    volumes:
      - ./inadyn.conf:/etc/inadyn.conf
    user: nobody:nogroup
```

**Comment by troglobit** [comment]
Thanks for the report.  Hopefully someone can verify and provide an idea of how to proceed, I don't use the docker build myself atm.  If nobody steps up I'll have a look at it right before the next release.

**Comment by leonyu** [comment]
Not really related to docker. Should probably be able to reproduce with the following (not tested): 
```
sudo -u nobody -g nogroup inadyn -F ./inadyn.conf --foreground --cache-dir=/tmp/ --no-pidfile
```
`./inadyn.conf` need to be readable by `nobody`

Unless cache-dir is at its default location (`/var/run/inadyn/`),  inadyn shouldn't try to chown the cache directory. The cache directory might be shared with other applications (e.g. /tmp), or maybe mapped from NFS or other containers with different uid/pid.

**Comment by troglobit** [comment]
Should be fixed as of a3616bf, please let me know if the problem persists.
